site/styles.css

```diff
@@ -526,6 +526,14 @@ figcaption {
   font-weight: bold;
 }
 
+.sidebar-item .menu-text {
```

Comment:
The CSS change for the text casing is a nice touch — how do you feel about making our other sidebar text a bit lighter for link texts as well? There’s something in our docs site that feels like it has too much contrast (text, navigation, or maybe a combination of both).

Reply:
Hm, I played around with it a bit (used a bit more of our green which is a lighter colour) but I don't know if it's too busy, though I do like the better contrast with the rest of the text:

https://github.com/validmind/documentation/blob/2c7be4f11513edaa5eb1d54bd96208b4f3e86f13/site/styles.css#L529

Reply:
I would stick with grey, maybe something halfway between the uppercased sidebar text and the current colour. 

Looking at preview, I think we'd need to apply this to both the sidebar navigation and right-hand ToC (or neither) for a consistent feel.

Reply:
I tried grey, and it looked off to me — too washed out. I would rather we revert to the original than use grey. I did open up a separate PR to make the ToC consistent though since this one was already merged before this comment: 

https://github.com/validmind/documentation/pull/524